ingest/src/server_lora.rs

```diff
@@ -65,7 +65,7 @@ impl poc_lora::PocLora for GrpcServer {
             .map_err(|_| Status::invalid_argument("invalid signature"))?;
 
         let event_id = EventId::from(&event);
-        match file_sink::write(&self.lora_beacon_report_tx, report).await {
+        match file_store::write!("beacon_report", &self.lora_beacon_report_tx, report).await {
```

Comment:
i would lean on keeping this macro/module naming consistent with the current function/method strategy, i.e. we read files from a file_store (s3 bucket) and write files to a file_sink (local file system that gets synced to s3 bucket out-of-band) to preserve the semantics of the two operational strategies

Reply:
Good call. I renamed the macro to `file_sink_write`. Deep namespacing doesn't seem possible with macros - you only get one level - crate.